listenbrainz/db/year_in_music.py

```diff
@@ -445,15 +450,19 @@ def notify_yim_users():
             "lb_logo_cid": lb_logo_cid[1:-1]
         }
 
-        send_mail(
-            subject="Year In Music 2021",
-            text=render_template("emails/year_in_music.txt", **params),
-            to_email=row["email"],
-            to_name=user_name,
-            html=render_template("emails/year_in_music.html", **params),
-            lb_logo_cid=lb_logo_cid,
-            lb_logo=lb_logo
-        )
+        try:
+            send_mail(
+                subject="Year In Music 2021",
+                text=render_template("emails/year_in_music.txt", **params),
+                to_email=row["email"],
+                to_name=user_name,
+                html=render_template("emails/year_in_music.html", **params),
+                lb_logo_cid=lb_logo_cid,
+                lb_logo=lb_logo
+            )
+        except Exception:
+            current_app.logger.error("Could not send YIM email", exc_info=True)
```

Comment:
will this include enough context information (in sentry?) for us to know who it was sent to?

Reply:
Sentry will capture local variable so user_name will be available in sentry to identify it.